### DISCUSSION TITLE
detectron2 cutstom dataset video detection error

### DISCUSSION THREAD

**Discussion by yasarniyazoglu** [facebookresearch/detectron2#3664] [Q&A]
I made segmentation with custom data. i got my weight and i test video but it doesn't detect anything. Weight and config file required for testing. i have weight but i don't know how to implement my own config file

`!git clone https://github.com/facebookresearch/detectron2
%run detectron2/demo/demo.py --config-file detectron2/configs/COCO-InstanceSegmentation/mask_rcnn_R_50_C4_3x.yaml --video-input /content/drive/MyDrive/v4_data/VID_20210510_085536695.mp4 --confidence-threshold 0.5 --output video-output.mp4 \
  --opts MODEL.WEIGHTS /content/output/model_final.pth`

how do i apply my own config to this code ?
--config-file detectron2/configs/COCO-InstanceSegmentation/mask_rcnn_R_50_C4_3x.yaml

**Comment by konstiantyn** [comment]
Same issue.
